koku/masu/test/processor/aws/test_aws_report_parquet_processor.py

```diff
@@ -96,11 +97,10 @@ def test_get_or_create_postgres_partition(self):
         bill_date = DateHelper().next_month_start
         pg_table = self.processor.postgres_summary_table._meta.db_table
         table_name = f"{pg_table}_{bill_date.strftime('%Y_%m')}"
-        expected_log = (
-            f"INFO:masu.processor.report_parquet_processor_base:"
-            f"Created a new partition for {pg_table} : {table_name}"
+        log_format = log_json(
+            msg="created a new partition", schema=self.schema_name, table=pg_table, partition=table_name
         )
-
+        expected_log = "INFO:masu.processor.report_parquet_processor_base:" + str(log_format)
```

Comment:
This assertion can be simplified to check for `created a new partition`. The other bits are inconsequential.

Reply:
```
======================================================================
FAIL: test_get_or_create_postgres_partition (masu.test.processor.aws.test_aws_report_parquet_processor.AWSReportProcessorParquetTest)
Test that a Postgres daily summary partition is created.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/cmyers/Projects/cost_mgmt/koku/koku/masu/test/processor/aws/test_aws_report_parquet_processor.py", line 107, in test_get_or_create_postgres_partition
    self.assertIn(expected_log, logger.output)
AssertionError: 'created a new partition' not found in ["INFO:masu.processor.report_parquet_processor_base:{'message': 'created a new partition', 'tracing_id': '', 'schema': 'org1234567', 'table': 'reporting_awscostentrylineitem_daily_summary', 'partition': 'reporting_awscostentrylineitem_daily_summary_2023_06'}", "INFO:masu.processor.report_parquet_processor_base:{'message': 'created a new partition', 'tracing_id': '', 'schema': 'org1234567', 'table': 'reporting_awscostentrylineitem_daily_summary', 'partition': 'reporting_awscostentrylineitem_daily_summary_2023_07'}", "INFO:masu.processor.report_parquet_processor_base:{'message': 'created a new partition', 'tracing_id': '', 'schema': 'org1234567', 'table': 'reporting_awscostentrylineitem_daily_summary', 'partition': 'reporting_awscostentrylineitem_daily_summary_2023_08'}"]
```